src/sqlalchemy/tables.py

```diff
@@ -9,6 +9,7 @@
     Column('id', UUID(as_uuid=True), primary_key=True, server_default=text("uuid_generate_v4()")),
     Column('username', String, unique=True, nullable=False),
     Column('email', String, unique=True, nullable=False),
+    Column('firebase_id', String, unique=True),
```

Comment:
```suggestion
    Column('firebase_id', String, unique=True, nullable=True),
```

Reply:
Even though this is not needed, I like being explicit about this because every part of a DB design is intentional and should be explicit.